translator/translate/otel/receiver/jmx/translator.go

```diff
@@ -41,8 +41,9 @@ const (
 )
 
 var (
-	configKey = common.ConfigKey(common.MetricsKey, common.MetricsCollectedKey, common.JmxKey)
-	localhost = collections.NewSet("localhost", "127.0.0.1")
+	configKey  = common.ConfigKey(common.MetricsKey, common.MetricsCollectedKey, common.JmxKey)
+	localhost  = collections.NewSet("localhost", "127.0.0.1")
+	jmxTargets = []string{"activemq", "cassandra", "hbase", "hadoop", "jetty", "jvm", "kafka", "kafka-consumer", "kafka-producer", "solr", "tomcat", "wildfly"}
```

Comment:
Can we centralize this into a single place? If we add support for more target systems, I want to make sure we don't accidentally only change it in the receiver and not the processor.